docs/release-notes/python-sdk.rst

```diff
@@ -0,0 +1,48 @@
+:orphan:
+
+**New Features**
+
+-  Python SDK: various new features and enhancements. A few highlights are listed below.
+      -  Add support for downloading a zipped archive of experiment code
+         (:meth:`Experiment.download_code
+         <determined.experimental.client.Experiment.download_code>`).
+
+      -  Add support for :class:`~determined.experimental.client.Project` and
+         :class:`~determined.experimental.client.Workspace` as SDK objects.
+
+      -  Surface more attributes to resource classes, including ``hparams`` and ``summary_metrics``
+         for :class:`~determined.experimental.client.Trial`.
+
+      -  Add support for filtering trial logs by timestamp and a query string using
+         :meth:`Trial.iter_logs <determined.experimental.client.Trial.iter_logs>`.
+
+      -  All resource objects now cache attributes by default and provide a ``.reload()`` method
+         which forces a fresh fetch.
+
+**Deprecated Features**
+
+-  Python SDK: several methods have been renamed for better API standardization.
+      -  ``list_*`` and ``iter_*`` for methods returning a ``List`` and ``Iterator``, respectively.
```

Comment:
Should I list these? I'd guess there are... 5? such methods in total.

Reply:
hmm well, we have deprecation warnings on everything, so you'd find out if you were using them in your code. i figured it might be very overwhelming if we just list all the methods we renamed, most of which probably won't be relevant to any individual user reading this.

but if there's only 5, that's not too bad. up to you, i'm ok either way.